### DISCUSSION TITLE
It works on a local network (LAN) without internet connection ?

### DISCUSSION THREAD

**Discussion by andrewssobral** [n0-computer/iroh#1905] [General]
Thank you guys for this nice library, congrats ! 
Just a simple question, it works on a local network (LAN) without internet connection ?

**Comment by b5** [comment]
Hey thanks! Local-only discovery will land as a default-on feature in the next minor version release, expected in early January. here’s the work in progress PR: https://github.com/n0-computer/iroh/pull/1914
